_posts/2018-07-19-futures-0.3.0-alpha.1.markdown

```diff
@@ -35,18 +35,34 @@ In other words:
 
 With futures 0.3 alpha, it is possible to use the new `async`/`await` notation that recently landed in nightly:
 
+```toml
+cargo-features = ["edition"]
+
+[package]
+edition = '2018
+```
+
 ```rust
+#![feature(async_await)]
+#![feature(await_macro)]
+
+extern crate futures;
```

Comment:
We're in Rust 2018 to get the `async` block. This should work without this line.